vercel-api/.github/workflows/ci.yml

```
@@ -0,0 +1,393 @@
+    - name: Wait for WordPress
+      run: |
+        timeout 300 bash -c 'until curl -f http://localhost:8080; do sleep 5; done'
+    
+    - name: Setup WordPress for testing
+      run: |
+        # Install WP-CLI
+        curl -O https://raw.githubusercontent.com/wp-cli/wp-cli/v2.8.1/utils/wp-completion.bash
+        curl -O https://raw.githubusercontent.com/wp-cli/wp-cli/v2.8.1/utils/wp-cli.phar
+        chmod +x wp-cli.phar
+        sudo mv wp-cli.phar /usr/local/bin/wp
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)